packages/@aws-cdk/aws-codebuild/lib/project.ts

```diff
@@ -1420,6 +1420,157 @@ export class LinuxBuildImage implements IBuildImage {
+    '1.15.2-gpu-py36-cu100-ubuntu18.04');
+  /** Tensorflow 2.0.0 GPU image from AWS Deep Learning Containers. */
+  public static readonly DLC_TENSORFLOW_2_0_0 = LinuxGpuBuildImage.DlcImage('tensorflow-training',
+    '2.0.0-gpu-py36-cu100-ubuntu18.04');
+  /** Tensorflow 2.0.1 GPU image from AWS Deep Learning Containers. */
+  public static readonly DLC_TENSORFLOW_2_0_1 = LinuxGpuBuildImage.DlcImage('tensorflow-training',
+    '2.0.1-gpu-py36-cu100-ubuntu18.04');
+  /** Tensorflow 2.1.0 GPU training image from AWS Deep Learning Containers. */
+  public static readonly DLC_TENSORFLOW_2_1_0_TRAINING = LinuxGpuBuildImage.DlcImage('tensorflow-training',
+    '2.1.0-gpu-py36-cu101-ubuntu18.04');
+  /** Tensorflow 2.1.0 GPU inference image from AWS Deep Learning Containers. */
+  public static readonly DLC_TENSORFLOW_2_1_0_INFERENCE = LinuxGpuBuildImage.DlcImage('tensorflow-inference',
+    '2.1.0-gpu-py36-cu101-ubuntu18.04');
+  /** Tensorflow 2.2.0 GPU training image from AWS Deep Learning Containers. */
+  public static readonly DLC_TENSORFLOW_2_2_0_TRAINING = LinuxGpuBuildImage.DlcImage('tensorflow-training',
+    '2.2.0-gpu-py37-cu101-ubuntu18.04');
+
+  /** PyTorch 1.2.0 GPU image from AWS Deep Learning Containers. */
+  public static readonly DLC_PYTORCH_1_2_0 = LinuxGpuBuildImage.DlcImage('pytorch-training',
+    '1.2.0-gpu-py36-cu100-ubuntu16.04');
+  /** PyTorch 1.3.1 GPU image from AWS Deep Learning Containers. */
+  public static readonly DLC_PYTORCH_1_3_1 = LinuxGpuBuildImage.DlcImage('pytorch-training',
+    '1.3.1-gpu-py36-cu101-ubuntu16.04');
+  /** PyTorch 1.4.0 GPU training image from AWS Deep Learning Containers. */
+  public static readonly DLC_PYTORCH_1_4_0_TRAINING = LinuxGpuBuildImage.DlcImage('pytorch-training',
+    '1.4.0-gpu-py36-cu101-ubuntu16.04');
+  /** PyTorch 1.4.0 GPU inference image from AWS Deep Learning Containers. */
+  public static readonly DLC_PYTORCH_1_4_0_INFERENCE = LinuxGpuBuildImage.DlcImage('pytorch-inference',
+    '1.4.0-gpu-py36-cu101-ubuntu16.04');
+  /** PyTorch 1.5.0 GPU training image from AWS Deep Learning Containers. */
+  public static readonly DLC_PYTORCH_1_5_0_TRAINING = LinuxGpuBuildImage.DlcImage('pytorch-training',
+    '1.5.0-gpu-py36-cu101-ubuntu16.04');
+  /** PyTorch 1.5.0 GPU inference image from AWS Deep Learning Containers. */
+  public static readonly DLC_PYTORCH_1_5_0_INFERENCE = LinuxGpuBuildImage.DlcImage('pytorch-inference',
+    '1.5.0-gpu-py36-cu101-ubuntu16.04');
+
+  /** MXNet 1.4.1 GPU image from AWS Deep Learning Containers. */
+  public static readonly DLC_MXNET_1_4_1 = LinuxGpuBuildImage.DlcImage('mxnet-training',
+    '1.4.1-gpu-py36-cu100-ubuntu16.04');
+  /** MXNet 1.6.0 GPU image from AWS Deep Learning Containers. */
+  public static readonly DLC_MXNET_1_6_0 = LinuxGpuBuildImage.DlcImage('mxnet-training',
+    '1.6.0-gpu-py36-cu101-ubuntu16.04');
+
+  /**
+   * @returns a Linux GPU build image from a Docker Hub image.
+   */
```

Comment:
> @skinny85 The only errors in the failing build are the following:
> 
> ```
> @aws-cdk/aws-codebuild: error: [awslint:docs-public-apis:@aws-cdk/aws-codebuild.LinuxGpuBuildImage.fromDockerRegistry] Public API element must have a docstring 
> @aws-cdk/aws-codebuild: error: [awslint:docs-public-apis:@aws-cdk/aws-codebuild.LinuxGpuBuildImage.fromEcrRepository] Public API element must have a docstring 
> ```
> 
> Both of these elements are already documented, though. I think there may be a bug in the linter.

The problem is that everything in your docs is inside the `@returns` tag, so the parser sees the actual body of the JSDoc for this method as empty. My suggestion (as the `@returns` tag is optional):

```suggestion
  /**
   * Returns a Linux GPU build image from a Docker Hub image.
   */
```